docs/blog/20240112-Daeraxa-JanuaryUpdate.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2024-01-12
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+Happy new year! Welcome to the first Pulsar community update of 2024!
+
+<!-- more -->
+
+# Welcome to the January Community Update!
+
+First of all, we hope everyone had a fantastic new year and here is to a new one! Things have been rather busy for the Pulsar team and the community at large, but we still have plenty of things to update you on!
```

Comment:
This doesn't make sense to me, but I may be missing something:

"Things have been busy ... but we still have plenty of things going on"

Was this meant to say things have been rather slow, or maybe "While the Pulsar team has been largely enjoying the holidays we still have plenty of things to update you on!"

Idk, maybe I'm just missing something here

Reply:
No I think this is tired brain re-writing stuff over and over.